final_prep/annotations.rb

```diff
@@ -3,11 +3,13 @@
 # Use the # to create a new comment
```

Comment:
@echon006 The directions specified adding an annotation for each line - I would highly encourage you to go back and do this in preparation for Mod 1.